dandi/cli/cmd_ls.py

```diff
@@ -60,9 +60,23 @@
     help="Convert metadata to new schema version",
     metavar="VERSION",
 )
+@click.option(
```

Comment:
Do you think @satra users should be given this opportunity?  I would prefer not to... Dear @jwodder , pleae make it a devel option which otherwise would not be visible.  Seems to be the first one for `ls` but we have them for `upload` etc.

Reply:
i'm fine with it being a DEVEL option.

Reply:
but then some kind of output should say "computing checksum" - the pyout output on drogon is not super helpful for me (number of rows displayed, etc.,.). so perhaps a status field could be added that gives such info in a column.

Reply:
displaying some message is orthogonal to having digest fake or not (that is what the option is about).

why are you interested to get that displayed BTW?    I guess we could add some `lgr.debug`  into get_digest for that purpose.  We have "mothballed for now" #465 to display digest computing progress during upload.  May be after/during Python API refactoring we could come up with some consistent interface across commands to display progress of long going operations.

Reply:
I've made the option devel-only and added an INFO-level log message just before calculating a digest.

Reply:
it's just that you have no idea as a user what the program is doing when showing ls. just like upload says validating, uploading, complete, it would be nice at least in the pyout display to know that something is going on.